src/test/java/tests/CampaignTest.java

```diff
@@ -0,0 +1,86 @@
+package tests;
+
+import api.ApiRequestManager;
+import api.ApiResponse;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import endpointurl.ElementParam;
+import endpointurl.Endpoint;
+import entities.Campaign;
+import entities.CreatedResponse;
+import entities.Product;
+import org.apache.http.HttpStatus;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CampaignTest extends CommonTest {
+
+    @BeforeMethod(onlyForGroups = {"post", "patch", "delete"})
+    public void createACampaign() throws JsonProcessingException {
+        Map<String, String> pathParams = new HashMap<>();
+        Campaign campaign = new Campaign();
+        campaign.setName("First campaign");
+        ApiResponse apiResponse;
+        apiResponse = ApiRequestManager.create(Endpoint.CAMPAIGNS, pathParams, campaign);
+        createdResponse = apiResponse.getResponse().as(CreatedResponse.class);
+        apiResponse.getResponse().then().assertThat().statusCode(HttpStatus.SC_CREATED);
+    }
+
+    @AfterMethod(onlyForGroups = {"post", "patch"})
+    public void deleteAProduct() {
+        Map<String, String> pathParams = new HashMap<>();
+        pathParams.put(ElementParam.ID, createdResponse.getId());
+        ApiResponse apiResponse;
+        apiResponse = ApiRequestManager.delete(Endpoint.CAMPAIGN, pathParams);
+        apiResponse.getResponse().then().assertThat().statusCode(HttpStatus.SC_NO_CONTENT);
+    }
+
+    @Test
+    public void getCampaign() {
+        Map<String, String> pathParams = new HashMap<>();
```

Comment:
The pathParams maps aren't used but are required in the ApiRequestManager, unneeded elements could probably be removed or hidden in the class.

Reply:
Already fixed